toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -507,6 +507,11 @@ func SystemBlockDevices() (systemDevices []SystemBlockDevice, err error) {
 		logger.Log.Warn(stderr)
 		return
 	}
+	if len(rawDiskOutput) == 0 {
+		err = fmt.Errorf("no supported disks found. Check lsblk output.")
```

Comment:
Does "supported" mean that there is a known list of disk types that are not supported?

I would also capitalize "no" -> "No" :)

Reply:
The detection is done by filtering the lsblk output by device major number. The currently supported block device/disk major numbers are enumerated in the const above. We can add to this list if needed.

Also golang code style is that the error string should not be capitalized. https://github.com/golang/go/wiki/CodeReviewComments#error-strings
I should remove the punctuation at the end though....

Reply:
Up to you, maybe that should go in the documentation: "We support these types of storage/We don't support these types of storage".

Also, thanks for the Go reference. Makes sense.

Reply:
Hmmm...looks like we could use some ISO installer documentation in general. Will work on that next and add info about the storage device types supported there.

Do note that the storage types supported by the ISO installer is different than the storage modules supported, which are determined by the kernel configs and loadable modules.